uint8 gpioId
uint8 edgeChangeType
uint32 timeSinceBoot_us 